docs/external-template-creation.md

```diff
@@ -84,3 +72,42 @@ export default {
 ```
 
 Once you've completed this step, your new sticker should now appear with the other stickers appended to the bottom of the shapes panel if the `enableStickers` feature flag is enabled.
+
+### Inserting stickers (SVGs) into a story
+
+To add stickers to your template, it's as easy as going to the experiments and enabling the `enableStickers` flag. Once turned on, all available stickers in the codebase should be available for selection under our base shapes in the shapes library panel.
+
+### Get the story JSON
+
+To get the raw story json, simply open up a story in the web story editor with the devtools network tab open and press `save`. You should see the the story's json show up in the network tab. In that json, right click the `story_data` field and click `Store as global variable`. Go to the devtools console and type 
+
+```javascript
+copy(temp1)
+```
+
+When you press enter the story data should now be copied to your clipboard and ready to paste where you need.
+
+**Note**: `Command+Shift+Option+J / Control+Shift+Alt+J` in the story editor opens up a dialog you can copy/paste raw story json to/from. However this is missing some fields `story_data` contains which are necessary for versioning the external templates.
+
+### Adding story JSON to the codebase as a new template
+
+Once you've added your template's raw story json to `packages/templates/src/raw/<template_name>.json`, you can import it into `packages/templates/src/getTemplates` and add to the function `loadTemplates(imageBaseUrl)`. It should then be accessible in `packages/templates/src/index`.
```

Comment:
There's no `loadTemplates` function.

Also, the template doesn't need to be imported individually.

```suggestion
Once you've added your template's raw story json to `packages/templates/src/raw/<template_name>.json`, you can add it to the list of templates in the `getTemplates()` function residing in `packages/templates/src/getTemplates`. It will then become accessible in the function exported by `packages/templates/src/index.js`, where you will need to add a new entry for the template (including its name, description, tags, etc.).
```